sorc/ncep_post.fd/GFSPOST.F

```diff
@@ -78,7 +78,7 @@ subroutine pvetc(km,p,px,py,t,tx,ty,h,u,v,av,hm,s,bvf2,pvn,theta,sigma,pvu)
   end subroutine
 ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
 ! - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-!> p2th interpolates to isentropic level.
+!> @brief p2th() interpolates to isentropic level.
```

Comment:
You don't need the brief tag on subroutines and functions. The first sentence is automatically known to be the brief description.

Only use the brief tag for files and module documentation blocks.

Reply:
Thanks! I made the changes.